tests/common/devices/multi_asic.py

```diff
@@ -535,6 +541,23 @@ def get_bgp_neighbors(self):
 
         return bgp_neigh
 
+    def get_bgp_neighbors_for_asic(self, namespace=None):
+        """
```

Comment:
Do we also want to consider the fact if the bgp neighbor state is `established` like the function `get_bgp_neighbors_per_asic`?

Btw this function looks similar to `get_bgp_neighbors_per_asic`. Could we leverage the same function to avoid repeating?

Reply:
I combined the code into get_bgp_neighbors() with correct logic. Now this works fine for multi-asic and non multi-asic scenario.